OpenEdXMobile/src/main/java/org/edx/mobile/model/api/CourseEntry.java

```diff
@@ -35,6 +36,12 @@
     private String discussion_url;
     private SocialURLModel social_urls;
     private CoursewareAccess courseware_access;
+    @Nullable private HashMap<String, String> course_sharing_utm_parameters;
+
+    @Nullable
+    public String getCourseSharingUtmParams(@NonNull String sharingPlatformKey) {
```

Comment:
This function should be at the bottom of file, respecting the already laid out format in file.